### DISCUSSION TITLE
[Feature Request] Template View Filters

### DISCUSSION THREAD

**Discussion by enoir3** [Syslifters/sysreptor#270] [General]
When working with a large number of template findings that are added to or modified often, it would be helpful if we were able to filter, based on current status, which templates appear in the main list. For example, if we are doing a team review, we could filter out all Templates that are not "Ready for review", or when individuals are working on improving Templates, they could filter the view to only show ones that are "In Progress" and/or "Needs Improvement".

I believe an argument could be made to add a filter for the severity or CVSS score, but that would be a plus instead of a functionality addition in this case.

**Comment by MWedl** [comment]
Good idea. We have an issue open for improving template search since quite a long time #75. However, we haven't started working on it yet. The best solution would be to add filters using text keywords in the searchbar (e.g. `status:in-progress`, similar to GitHub) and provide autocomplete dropdowns for these keywords. 

Adding separate dropdowns for tags, status, severity, etc. would bloat the UI and, so will not be considering this solution.

**Reply by enoir3** [reply]
I understand the bloating aspect. Text keywords is also a good idea, especially since a search function has already been implemented. Thank you for the feedback and the bump on your existing topic :)

**Comment by aronmolnar** [comment]
This is currently being implemented in #470. In this ticket, you see what filters we currently implemented. If you have more requirements, please tell us :)

Closing this as a duplicate.
